extensions/jupyter-adapter/src/LanguageRuntimeAdapter.ts

```diff
@@ -146,7 +146,7 @@ export class LanguageRuntimeSessionAdapter
 		let response = {} as any;
 		try {
 			// Send the request and wait for a response
-			response = await uiComm.performRpc(request);
+			response = (await uiComm.performRpc(request)).data;
```

Comment:
This pattern is repeated a few times. How about having a `performRpc` variant that just returns the `data` portion of the message?